src/components/new-suggestion/areas/review.area.tsx

```diff
@@ -125,15 +78,18 @@ const ReviewArea = () => {
                   imageURL={instrument.instrument.image_source}
                   name={instrument.instrument.instrument_name}
                   description={instrument.description}
-                  uid={uid}
+                  uid={undefined}
```

Comment:
why undefined?

Reply:
Only option usable for the usecase I had. reviewSection requires no uid for review area so i used the one the interface supported

Reply:
Can't you just make the prop nullable tho?